### PR TITLE
fix(frontend, kanban): do not use the label as filename.

### PR DESCRIPTION
**Pull request by grignards**
The label is not necessary for kanban boards and the GET request fails if a double-quote
character is part of the label.

Issue Ref: #5220

-------

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by gbonaspetti — APPROVED** [review]

File: frontend_app_kanban/src/component/Kanban.jsx [hunk 1]
```diff
@@ -90,23 +90,27 @@ export class Kanban extends React.Component {
...
-      })
-    } else {
+    try {
+      const fetchRawFileContent = await handleFetchResult(
+        await getRawFileContent(
+          props.config.apiUrl,
+          props.content.workspace_id,
+          props.content.content_id,
+          props.content.current_revision_id,
+          'kanban' + KANBAN_FILE_EXTENSION
```
Comment (gbonaspetti) [inline frontend_app_kanban/src/component/Kanban.jsx:100]: a const for `'kanban'`might be good, but not required

Comment (grignards) [reply frontend_app_kanban/src/component/Kanban.jsx]: Done.

**Review by Skylsmoi — APPROVED** [review]

**Comment by gbonaspetti** [comment]
When editing the title, it works well, but if I create a kanban with double quotes the title is shown like this:
Screenshot from 2021-12-31 11-18-26

**Comment by grignards** [comment]
> * if I edit a title, it works well, until I made another modif (see gif)
>       quotes
> 
>     * if I create a kanban with double quotes the title is shown like this:
>       Screenshot from 2021-12-31 11-18-26

Both are due to this issue: https://github.com/tracim/tracim/issues/4015 + the fact that the browser url-encodes the filename but the backend does not decode them, I just created an issue for this bug: https://github.com/tracim/tracim/issues/5228

**Comment by PhilippeAccorsi** [comment]
@lebouquetin maybe ok for the 4.7.0 ?

**Comment by Skylsmoi** [comment]
tested on branch

**Pull request merged**